### PR TITLE
Update app pages

### PR DESCRIPTION
**Pull request by MahmoudHafezDarwish**

### CODE REVIEW

File: lib/core/remote_config/page/soft_update_page.dart [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+import 'package:flutter_svg/flutter_svg.dart';
+
+class SoftUpdatePage extends StatelessWidget {
+  const SoftUpdatePage({Key? key, this.message}) : super(key: key);
+
+  final String? message;
+
+  @override
+  Widget build(BuildContext context) {
+    return SizedBox(
```
Comment (salahamassi) [inline lib/core/remote_config/page/soft_update_page.dart:15]: remove size box

File: lib/core/remote_config/page/soft_update_page.dart [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                SvgPicture.asset(
+                  AppIcons.forceUpdate,
+                  width: 104,
+                  height: 104,
+                ),
+                const SizedBox(height: 32),
+                const Text(
+                    // context.localizations.update_app ?? '',
```
Comment (salahamassi) [inline lib/core/remote_config/page/soft_update_page.dart:42]: use context.localizations ?

File: lib/core/remote_config/page/update_app_page.dart [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+import 'package:bond/core/app_widgets.dart';
+import 'package:bond/core/resources/app_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+
+
+class UpdateAppPage extends StatefulWidget {
+  final String? message;
+  static var shown = false;
```
Comment (salahamassi) [inline lib/core/remote_config/page/update_app_page.dart:11]: delete this variable

File: lib/core/remote_config/page/update_app_page.dart [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+              ),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  void _onUpdate() {
+    //TODO::CHANGE appStoreId
```
Comment (salahamassi) [inline lib/core/remote_config/page/update_app_page.dart:89]: use any apple app id

File: lib/core/remote_config/page/update_app_page.dart [hunk 5]
```diff
@@ -0,0 +1,95 @@
+import 'package:bond/core/app_localizations.dart';
+import 'package:bond/core/app_widgets.dart';
+import 'package:bond/core/resources/app_icons.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+
+
+class UpdateAppPage extends StatefulWidget {
+  final String? message;
```
Comment (salahamassi) [inline lib/core/remote_config/page/update_app_page.dart:10]: why message is nullable?

File: lib/core/remote_config/page/soft_update_page.dart [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+import 'package:bond/core/app_localizations.dart';
+import 'package:bond/core/resources/app_icons.dart';
+import 'package:bond/core/widgets/app_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class SoftUpdatePage extends StatelessWidget {
+  const SoftUpdatePage({Key? key, this.message}) : super(key: key);
+
+  final String? message;
```
Comment (salahamassi) [inline lib/core/remote_config/page/soft_update_page.dart:11]: why the message is nullable?

File: lib/features/post/data/datasource/post_remote_data_source.dart [hunk 7]
```diff
@@ -29,7 +29,7 @@ class PostRemoteDataSource extends DataSource {
 }
 
 extension PostsApi on Api {
-  static String get posts => 'posts?term=women';
+  static String get posts => 'posts?term=man';
```
Comment (salahamassi) [inline lib/features/post/data/datasource/post_remote_data_source.dart:32]: sky

File: lib/core/remote_config/page/soft_update_page.dart (hunk 1 above)
Comment (MahmoudHafezDarwish) [reply lib/core/remote_config/page/soft_update_page.dart:15]: done

File: lib/core/remote_config/page/soft_update_page.dart (hunk 2 above)
Comment (MahmoudHafezDarwish) [reply lib/core/remote_config/page/soft_update_page.dart:42]: done

File: lib/core/remote_config/page/update_app_page.dart (hunk 5 above)
Comment (MahmoudHafezDarwish) [reply lib/core/remote_config/page/update_app_page.dart:10]: done

File: lib/core/remote_config/page/soft_update_page.dart (hunk 6 above)
Comment (MahmoudHafezDarwish) [reply lib/core/remote_config/page/soft_update_page.dart:11]: I make it required

File: lib/features/post/data/datasource/post_remote_data_source.dart (hunk 7 above)
Comment (MahmoudHafezDarwish) [reply lib/features/post/data/datasource/post_remote_data_source.dart:32]: doen

**Pull request merged**